projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -217,6 +211,12 @@ export class PdfViewerComponent implements AfterContentInit, OnChanges, OnDestro
     this.setPageHeights();
   }
 
+  private setDocumentProgress() {
```

Comment:
could we maybe change the name of this function to initDocumentProgress to make it clear how it's related to the other function onDocumentLoadProgress

Reply:
Renamed to initDocumentProgress as per recommendation